cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/unmanaged-cluster-wizard-page/unmanaged-cluster-network-settings-step/UnmanagedClustersNetworkSettings.tsx

```diff
@@ -0,0 +1,172 @@
+                                        ></input>
+                                        {errors['CLUSTER_SERVICE_CIDR'] && (
+                                            <CdsControlMessage status="error">{errors['CLUSTER_SERVICE_CIDR'].message}</CdsControlMessage>
+                                        )}
+                                    </CdsInput>
+                                </div>
+                                <div cds-layout="col:4">
+                                    <CdsInput layout="vertical" control-width="shrink">
+                                        <label cds-layout="p-b:md">
+                                            Cluster pod CIDR <CdsIcon shape="info-circle" size="md" status="info"></CdsIcon>
+                                        </label>
+                                        <input
+                                            {...register('CLUSTER_POD_CIDR')}
+                                            placeholder="CLUSTER POD CIDR"
+                                            onChange={handleClusterPodCidrChange}
+                                            defaultValue="100.96.0.0/11"
+                                        ></input>
+                                        {errors['CLUSTER_POD_CIDR'] && (
+                                            <CdsControlMessage status="error">{errors['CLUSTER_POD_CIDR'].message}</CdsControlMessage>
+                                        )}
+                                    </CdsInput>
+                                </div>
+                            </div>
+                            <div cds-layout="col:4">
+                                <CdsInput>
+                                    <label cds-layout="p-b:md">
+                                        Node to host port mapping <CdsIcon shape="info-circle" size="md" status="info"></CdsIcon>
+                                    </label>
+                                    <input
+                                        {...register('CLUSTER_NAME')}
+                                        placeholder="Cluster name"
+                                        onChange={handleClusterNameChange}
+                                        defaultValue={'127.0.0.1:80:80/tcp'}
+                                    ></input>
+                                    {errors['CLUSTER_NAME'] && (
+                                        <CdsControlMessage status="error">{errors['CLUSTER_NAME'].message}</CdsControlMessage>
+                                    )}
+                                    <CdsControlMessage className="description" cds-layout="m-t:sm">
+                                        Can only contain lowercase alphanumeric characters and dashes.
+                                        <br></br>
+                                        <br></br>
+                                        The name will be used to reference your cluster in the Tanzu CLI and kubectl.
```

Comment:
IMHO, This message seems a bit out of place right next to `Node to host port mapping` section.

Reply:
You are right that is not the right message I forgot to change it. Thank you!